.pre-commit-hooks.yaml

```diff
@@ -1,10 +1,28 @@
+- id: codespell
+  name: codespell
+  entry: codespell --check-filenames --ignore-words-list READYness
```

Comment:
Let's move this to `pyproject.toml` to support people directly invoking `codespell` too:

```toml
[tool.codespell]
check-filenames = true
check-hidden = true
ignore-words-list = "READYness"
```

Reply:
I probably should turn on check-hidden to cover the .github directory. That does need a corresponding exclude file though. Hopefully there's a way to reuse .gitignore.

Unfortunately if we moved the configuration to pyprojet.toml, other projects/repositories like https://github.com/biobuddies/allowedflare would need to copy and paste that configuration fragment. I guess Prettier and Ruff have the same problem.

Maybe .pre-commit-hooks.yaml could have a minimum configuration and pyproject.toml could have a full one?